### PR TITLE
feat(dto): export `ThirdpartyRequestsVerifications*` types 

### PR DESCRIPTION
**Pull request by lewisdaly**
Is this the right way to do it? I noticed that these weren't added to the `index.ts` automatically.

### CODE REVIEW

**Comment by kleyow** [comment]
@lewisdaly Yep. Add a test though just to make sure it's being exported and generated in https://github.com/mojaloop/api-snippets/blob/master/tests/dto/thirdparty.test.ts

**Review by kleyow — CHANGES_REQUESTED** [review]
@lewisdaly Yep. Add a test though just to make sure it's being exported and generated in https://github.com/mojaloop/api-snippets/blob/master/tests/dto/thirdparty.test.ts

File: tests/dto/thirdparty.test.ts [hunk 1]
```diff
@@ -561,6 +561,29 @@ describe('thirdparty', () => {
...
+    expect(thirdpartyRequestsVerificationsIdPutResponse).toBeDefined()
+  })
+
+  test('ThirdpartyRequestsVerificationsPostRequest', () => {
+    const thirdPartyRequestsVerificationsPostRequest: Schemas.ThirdpartyRequestsVerificationsPostRequest = {
+      verificationRequestId: correlationId,
+      challenge: 'some challenge base64 encoded',
+      value: {
+        authenticationInfo: {
+          authentication: 'U2F',
```
Comment (kleyow) [inline tests/dto/thirdparty.test.ts:577]: I think this request body needs to be updated to handle `clientDataJson`/`authenticatorData`/`signature`/`userHandle` https://webauthn.guide/#authentication

**Review by kleyow — APPROVED** [review]

**Review by kleyow — CHANGES_REQUESTED** [review]
https://github.com/mojaloop/api-snippets/pull/99/files 
Totally forgot that I started addressing those changes here.

**Comment by kleyow** [comment]
Once we settle on the schema changes I have the updates to the `central-ledger` and `sdk-standard-components` standing by.

**Comment by lewisdaly** [comment]
> https://github.com/mojaloop/api-snippets/pull/99/files
> Totally forgot that I started addressing those changes here.

Yeah I was wondering about that... do you want to merge that PR in first, then we can do mine?

**Comment by kleyow** [comment]
Sounds good. It's building now.

**Review by kleyow — APPROVED** [review]

File: tests/dto/thirdparty.test.ts (hunk 1 above)
Comment (kleyow) [reply tests/dto/thirdparty.test.ts:597]: Not sure how we want to change this payload to include `FIDOPublicKeyCredentialAssertion` but lets do that in another PR.

**Pull request merged**